src/views/SystemIntake/ContactDetails/index.tsx

```diff
@@ -58,11 +58,11 @@ export type ContactDetailsForm = {
 };
 
 type ContactDetailsProps = {
-  formikRef: any;
   systemIntake: SystemIntakeForm;
 };
 
-const ContactDetails = ({ formikRef, systemIntake }: ContactDetailsProps) => {
+const ContactDetails = ({ systemIntake }: ContactDetailsProps) => {
+  const formikRef: any = useRef();
```

Comment:
```suggestion
  const formikRef = useRef<FormikProps<ContactDetailsForm>>();
```

Can you check if this works here?